src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -174,7 +174,7 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*OVClass.*QueryNetwork.*QueryNetwork(HETEROWithDeviceIDNoThrow|WithDeviceID|WithBigDeviceIDThrows|WithInvalidDeviceIDThrows|HETEROWithBigDeviceIDThrows).*)",
         R"(.*OVClass.*LoadNetwork.*(DeviceID|MultiWithoutSettingDevicePrioritiesThrows).*)",
         R"(.*OVClassLoadNetworkTest.*QueryNetwork(MULTIWithHETERO|HETEROWithMULTI)NoThrow_V10.*)",
-        R"(.*OVClassNetworkTestP.*QueryNetworkMultiThrows.*)",
+        R"(.*OVClassNetworkTestP.*QueryNetworkMultiNoThrows.*)",
```

Comment:
Why test is renamed? Actually we expect that it throws

Reply:
And also I'm waiting that this line will be removed from skiped list)

Reply:
@nkogteva aa, ok, I misunderstood, I will, but QueryNetworkMultiNoThrows is continue to fail, just  LoadNetworkMultiWithoutSettingDevicePrioritiesThrows is fixed. As QueryNetworkMultiNoThrows fails should I remove it anyway ?

Reply:
> @nkogteva aa, ok, I misunderstood, I will, but QueryNetworkMultiNoThrows is continue to fail, just LoadNetworkMultiWithoutSettingDevicePrioritiesThrows is fixed. As QueryNetworkMultiNoThrows fails should I remove it anyway ?

For QueryNetwork fix from this PR https://github.com/openvinotoolkit/openvino/pull/14434/ should help. QN should not raise an exception. Please check

Reply:
ok, got it, thank you, tests are removed from skip configs